app/javascript/review.js

```diff
@@ -21,6 +21,19 @@ route(/\/review\/[\w-]+\/?\d*$/i, async () => {
     $('.review-item-container').html(html);
 
     installSelectpickers();
+
+    if (queuePath.includes('untagged-domains')) {
+      const taggifyInputs = document.getElementsByName('tag_name');
+      for (let i = 0; i < taggifyInputs.length; i++) {
+        window.Taggify.taggify_element(taggifyInputs[i]);
+        taggifyInputs[i].parentElement.addEventListener('tag_change', function () {
```

Comment:
```js
$(taggifyInputs[i]).parent().on('tag_change', ev => { ... });
```

Reply:
This one I'm keeping as is because IIRC I was getting into a fight with jQuery about this.